src/LanguageServer/Protocol/Features/Diagnostics/EngineV2/DiagnosticIncrementalAnalyzer_GetDiagnosticsForSpan.cs

```diff
@@ -43,6 +43,31 @@ public async Task<ImmutableArray<DiagnosticData>> GetDiagnosticsForSpanAsync(
             return list.ToImmutableAndClear();
         }
 
+        private static async Task<ImmutableDictionary<DiagnosticAnalyzer, ImmutableArray<DiagnosticData>>> ComputeDocumentDiagnosticsCoreAsync(
```

Comment:
these are moves of a couple of static methods to an outer scope so they can be seen by a few nested types.